src/mopi/blocks/transformations/lemmatizer.py

```diff
@@ -20,7 +20,7 @@ def __init__(self, remove_stopwords: bool) -> None:
         self.remove_stopwords = remove_stopwords
 
     def load(self) -> None:
-        self.nlp = get_spacy()
+        get_spacy()
```

Comment:
We aren't accessing ``self.nlp`` anywhere here so it is better not to store the object.